test/blocks/global-footer/global-footer.test.js

```diff
@@ -272,4 +272,70 @@ describe('global footer', () => {
       }
     });
   });
+
+  describe('LANA logging tests', () => {
+    beforeEach(async () => {
+      window.lana.log = sinon.spy();
```

Comment:
Should this be restored in an `afterEach`?

Reply:
There is a `sinon.restore();` on L45 which will take care of that for all tests in the footer.